web/app_options.js

```diff
@@ -143,6 +143,11 @@ const defaultOptions = {
     value: typeof PDFJSDev === "undefined" || PDFJSDev.test("TESTING"),
     kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
   },
+  enableML: {
+    /** @type {boolean} */
+    value: false,
+    kind: OptionKind.PREFERENCE,
```

Comment:
Given that "PREFERENCE" is a slightly special option-kind, we should always ensure that each option has one of the "regular" kinds as well.
(Actually, as it happens, I've got a PR in review that enforces this.)
```suggestion
    kind: OptionKind.VIEWER + OptionKind.PREFERENCE,
```